### ISSUE TITLE
I can't do az login in Powershell.

### ISSUE THREAD

**Issue by AnNamGi** [Azure/azure-cli#25448]
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62

**Related command**
 
![image](https://user-images.githubusercontent.com/81275109/218613346-1c9419b4-7735-49ab-80bd-df279030a79a.png)

**Describe the bug**

Opening the power shell with administrator privileges and running Azlogin does not work as a permission issue.

**Environment summary**
<!--- Install Method (e.g. pip, interactive script, apt-get, Docker, MSI, edge build) / CLI version (`az --version`) / OS version / Shell Type (e.g. bash, cmd.exe, Bash on Windows) 
Windows 10 Home

**Comment by yonzhan** [comment]
@jiasli for awareness

**Comment by jiasli** [comment]
Sorry that I can't understand Korean. 

- Could you share the error message in text, instead picture, so that I can copy the message to a translator.
- Or, could you change your system language to English and share the error message in English?
- Also, what error message you get if you run `az login` in Command Prompt?

**Comment by AnNamGi** [comment]
here this
-----Original Message-----

**Comment by MoChilia** [comment]
Hi @AnNamGi! The information you replied is missing via email. Could you kindly leave comments on GitHub: [Issue #25488](https://github.com/Azure/azure-cli/issues/25448)?

**Comment by AnNamGi** [comment]
![image](https://user-images.githubusercontent.com/81275109/218690168-e5fe1dc2-b036-4146-926b-11d2d8f06c0c.png)
Here you are!

**Comment by AnNamGi** [comment]
GitHub: Issue #25488 Posted in the comments!
-----Original Message-----

**Comment by MoChilia** [comment]
@AnNamGi, have you tried to run PowerShell as administrator?

**Comment by AnNamGi** [comment]
Yes, I tried to run it with administrator privileges, but it doesn't work.

Also, I used the administrator account, but it doesn't work.

**Comment by MoChilia** [comment]
@AnNamGi, could you share
1. Which version of Windows operating system, PowerShell and Azure CLI you are using?
2. How did you install Azure CLI?
3. Have you installed any antivirus? Try to temporary turn off them.
4. Does the command `az` work in Command Prompt?

**Comment by AnNamGi** [comment]
1. Which version of Windows operating system are you using, PowerShell or Azure CLI? Windows 10 Home

2. How did you install Azure CLI?
I installed it by pressing the red square box part of the image below on https://learn.microsoft.com/ko-kr/cli/azure/install-azure-cli-windows?tabs=azure-cli 
![image](https://user-images.githubusercontent.com/81275109/218952861-651a7421-255a-41be-9fd9-f92f682b0675.png)

3. Have you installed an antivirus? Temporarily turn off the power.
Reason Cybersecurity is turned on, temporarily powered off, and then attempts az login in the power shell, but it does not work. 

4. Does the az command work at the command prompt? The az command works at the command prompt.

5. What is the output of Get-Command z | Format-List?
![image](https://user-images.githubusercontent.com/81275109/218952931-5bc0373d-a16d-471b-ac62-d673ed759169.png)

**Comment by MoChilia** [comment]
Thanks @AnNamGi. It seems like a permission issue specific to PowerShell. Could you tell us more about what's the version of **PowerShell** and **Azure CLI** you are using and try the latest [PowerShell 7.3](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-7.3) and [Azure CLI 2.45.0](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli)?

**Comment by AnNamGi** [comment]
![2023-02-16 09 14 15](https://user-images.githubusercontent.com/81275109/219225126-e2f29921-a7dc-4470-b193-28c3f9f847eb.png)
It doesn't work even with the latest version of PowerShell.

**Comment by jiasli** [comment]
I did some search and found this issue happens to other programs as well, not only limited to Azure CLI:

- https://stackoverflow.com/questions/69671055/program-a-exe-failed-to-run-access-is-deniedat-line1-char1
- https://stackoverflow.com/questions/65639505/program-tempcoderunnerfile-exe-failed-to-run-access-is-deniedat-line1-char1
- https://discuss.hashicorp.com/t/program-terraform-exe-failed-to-run-access-is-deniedat-line-1-char-1/30316
- https://github.com/microsoft/winget-cli/issues/876

However, there doesn't seem to be any reliable solution that can solve it.

**Comment by AnNamGi** [comment]
![2023-02-16 13 15 57](https://user-images.githubusercontent.com/81275109/219267544-f9fa01a4-59ae-4cab-8a27-e8d462ebd10e.png)
Az.cmd is verified to have permission to execute.

It is disappointing that there is no solution.

I downgraded from Windows 11 to Windows 10, and I remember that it worked az.cmd on Windows 11.

How about upgrading to Windows 11?

**Comment by jiasli** [comment]
If you create a new cmd like `D:\test.cmd` and put some commands in it:

```cmd
echo "Hello World"
```

Then can you run it in PowerShell?

```
& D:\test.cmd
```

**Comment by AnNamGi** [comment]
I did as you said, but the same authority error occurs again.
